packages/itwinui-react/src/core/Table/columns/expanderColumn.tsx

```diff
@@ -63,6 +63,7 @@ export const ExpanderColumn = <T extends Record<string, unknown>>(
       } else {
         return (
           <IconButton
+            aria-label='Toggle expandable content'
```

Comment:
These labels should ideally be unique, so that assistive technology users are able to differentiate between each instance more easily. Typically, this is done by combining two labels, i.e. "Toggle expandable content, [name of the row]".

How do we handle this for regular expandable sub-rows?

Reply:
Ok, then let's go with this for now, since it's better than nothing. But we should improve it in the future.